bin/spaid_setup_dev_workspace

```diff
@@ -7,21 +7,35 @@ if [[ "$1" == "-h" ]] ; then
     echo "workspace to have any helper packages or tools installed and"
     echo "configured. These are the actions it does currently:"
     echo 
-    echo "- Installs Python package dependencies"
+    echo "- Installs Python package dependencies (if in a Poetry project)"
     echo "- Installs pre-commit hooks"
     echo "- Runs pre-commit hooks on all the files"
     echo 
     echo "This command only works if:"
     echo 
     echo "- you are in a local Git repository"
-    echo "- the repository is a Python project managed by Poetry"
     echo "- the repository has a file called '.pre-commit-config.yaml'"
     exit 0
 fi
 
-# Install dependencies just in case.
-poetry install
+precommit_file=$(pwd)/.pre-commit-config.yaml
+pyproject_file=$(pwd)/pyproject.toml
```

Comment:
Should this variable be used somewhere later?

Reply:
ooops, missed that! Thanks :star_struck: